app/static/currency.js

```diff
@@ -0,0 +1,71 @@
+function getCurrency(baseCurrency, dateToday) {
```

Comment:
You should be proud of yourself for this code. Very well done

Reply:
I learned so much, thank you so much for the feedback!